src/ORM/FieldType/DBString.php

```diff
@@ -17,13 +24,14 @@ abstract class DBString extends DBField
     ];
 
     /**
-     * Set the default value for "nullify empty"
+     * Set the default value for "nullify empty" and 'default'
      *
      * {@inheritDoc}
      */
     public function __construct($name = null, $options = [])
     {
         $this->options['nullifyEmpty'] = true;
+        $this->options['default'] = '';
```

Comment:
Why use `options` for this instead of `setDefaultValue()`?

Reply:
There's existing code in setOptions() that calls setDefaultValue()

Reply:
Seems like we should remove that and just call `setDefaultValue()` instead for consistency but I'll let it be.